server/routes/api/checkout.js

```diff
@@ -42,48 +42,56 @@ router.post('/create-transaction', async (req, res) => {
 //  a Stripe session_id included in the URL.
 
 router.post('/create-checkout-session', async (req, res) => {
-  try {
-    const acceptableCharges = [1, 2, 20, 50]
-    const { donationAmount } = req.body || {}
-    const parsedDonationAmount = parseInt(donationAmount, 10)
+  const { donationAmount } = req.body || {}
+  const origin = req.get('origin')
 
-    let donation
+  const input = formatDonationAmount(donationAmount)
+  const inputIsValid = validateDonationAmount(input)
+  const donationAmountForStripe = input * 100 // Stripe accepts values in cents
```

Comment:
You could move this into the try block and save a calculation. Not a big deal, however.

Reply:
Agreed. It's an easy fix though.

Reply:
Actually, I'm thinking that it makes sense to move the computation inside of the `if` statement, not the `try` block. How would moving this into the `try` block save a calculation?

Or did you mean that creating a const donationAmountForStripe is not necessary? Again, readability was the only reason, and it'd also be a super easy fix.

Reply:
You are correct to move it inside the `if` block. I was thinking that it was a little tighter grouping inside the `try`, but now that I see what you did, I like that better.

I believe that `donationAmountForStripe` is necessary because we have to send the donation value in cents to the Stripe API.